tests/test_modeling_tf_common.py

```diff
@@ -28,6 +28,7 @@
 from requests.exceptions import HTTPError
 from transformers import is_tf_available
 from transformers.models.auto import get_values
+from transformers.testing_utils import tooslow  # noqa: F401
```

Comment:
Why not leave it in the imports below?

Reply:
This caused style issues because of the #noqa tag

Reply:
But if it's not used anymore, why note remove it, I'm confused.

Reply:
Ah, I'm sorry! I thought it should stay imported because it might be needed in that file, but that's probably a stupid way to do things. Removing it!